File: 3462-vowels-game-in-a-string/3462-vowels-game-in-a-string.js
/**
 * @param {string} s
 * @return {boolean}
 */
var doesAliceWin = function (s) {
    s = s.split('')
    count = 0
    for (let i = 0; i < s.length; i++) {
        if (s[i] == 'a' || s[i] == 'e' || s[i] == 'i' || s[i] == 'o' || s[i] == 'u') {
            count++
        }
    }
    if (count > 0) {
        return true
    }
    else {
        return false
    }
};

File: 3462-vowels-game-in-a-string/3462-vowels-game-in-a-string.py
class Solution:
    def doesAliceWin(self, s: str) -> bool:
        s = list(s)
        count = 0
        for i in range(len(s)):
            if s[i] == 'a' or s[i] == 'e' or s[i] == 'i' or s[i] == 'o' or s[i] == 'u':
                count+=1
        if count > 0:
            return True
        else:
            return False



